src/docs/concepts-development/content-management/index.md

```diff
@@ -0,0 +1,91 @@
+---
+title: Content management in Flex
+slug: content-management
+updated: 2022-07-28
+category: concepts-development
+ingress:
+  This article introduces the Flex content management system and how
+  static pages can be managed in Flex Console
+published: true
+---
+
+In 2022-09, Flex introduces a headless CMS-like feature to create static
+pages in Flex. Initially, the feature allows operators to modify their
+landing pages without code. Going forward, it will be possible to create
+new pages as well.
+
+## Content modeling in Flex
+
```

Comment:
TODO: Glossary of terms

Reply:
Could be:

Page Builder
Content modelling
Content management
Page schema
Assets
Asset Delivery API
Page
Sections
Blocks
Fields
Page asset data